### ISSUE TITLE
크로스엔트로피_파이토치

### ISSUE THREAD

**Issue by nureeee** [nureeee/DeepLearning#13]
## Cross-entropy Loss 
torch.nn.functional

> torch.log(F.softmax(z, dim=1))
> F.log_softmax(z, dim=1)

같은 결과를 도출한다
